akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -320,19 +331,31 @@ private[akka] object Running {
 
     def onSnapshotterResponse(response: SnapshotProtocol.Response): Unit = {
       response match {
-        case SaveSnapshotSuccess(meta) ⇒
+        case e @ SaveSnapshotSuccess(meta) ⇒
+          setup.log.debug(s"Persistent snapshot [{}] saved successfully", meta)
+          internalDeleteEvents(e, state)
           setup.onSnapshot(meta, Success(Done))
-        case SaveSnapshotFailure(meta, ex) ⇒
-          setup.onSnapshot(meta, Failure(ex))
 
-        // FIXME #24698 not implemented yet
-        case DeleteSnapshotFailure(_, _)  ⇒ ???
-        case DeleteSnapshotSuccess(_)     ⇒ ???
-        case DeleteSnapshotsFailure(_, _) ⇒ ???
-        case DeleteSnapshotsSuccess(_)    ⇒ ???
+        case SaveSnapshotFailure(meta, e) ⇒
+          setup.log.warning(
+            "Failed to saveSnapshot given metadata [{}] due to: [{}: {}]",
+            meta, e.getClass.getCanonicalName, e.getMessage)
+          setup.onSnapshot(meta, Failure(e))
+
+        case DeleteSnapshotSuccess(meta) ⇒
+          setup.log.debug(s"Persistent snapshot [{}] deleted successfully", meta)
+
+        case DeleteSnapshotsSuccess(meta) ⇒
+          setup.log.debug(s"Persistent snapshots [{}] deleted successfully", meta)
+
+        case DeleteSnapshotFailure(meta, e) ⇒
+          setup.log.warning("Failed to deleteSnapshot given metadata [{}] due to: [{}: {}]", meta, e.getClass.getCanonicalName, e.getMessage)
+
+        case DeleteSnapshotsFailure(c, e) ⇒
+          setup.log.warning("Failed to deleteSnapshots given criteria [{}] due to: [{}: {}]", c, e.getClass.getCanonicalName, e.getMessage)
```

Comment:
These are covered by #26484 so if we want defaults for these it should go in the default signal handler after that is merged.